app/assets/js/app/modules/household.js

```diff
@@ -30,9 +30,9 @@ class HouseholdMember extends EventEmitter {
       }
     }
 
-    this.inputs = this.node.querySelectorAll('input')
+    this.inputs = this.node.querySelectorAll('.js-btn-remove')
```

Comment:
`this.inputs` is used elsewhere, this will break it.

in the `bindToExisting` method, there's already a reference to the removeBtn—just do the attribute thing there